tests/no-inheritance-anon-class.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Test fix for #659
+--DESCRIPTION--
+Unbound anon class causing segfaults, we delay copy but still cannot serialize the anon
+--FILE--
+<?php
+$task = new class extends Thread {
+    public function run()
+    {
+        $this->prop = new class {};
+		var_dump($this->prop); /* we do expect null: anon classes cannot be serialized */
```

Comment:
I'd say this behaviour is entirely expected. When an object of an unserialisable class is assigned to a property of a `Threaded` object, it will assign `null` instead. This can already be seen with the `SimpleXMLElement`, `HashContext`, and `SplFileInfo` classes, for example. 

Whether these semantics should be changed (to throw an exception, for example) is orthogonal to this PR.